packages/maskbook/src/locales/ko-KR.json

```diff
@@ -508,6 +508,10 @@
     "plugin_ito_amount_unlocked": "{{amount}} {{symbol}} 언락됨",
     "plugin_ito_amount_unlocked_infinity": "언락된 인피니티 {{symbol}}",
     "plugin_ito_unlocking_symbol": "{{symbol}} 언락 중",
+    "plugin_ito_continue": "",
```

Comment:
Don't add those empty string because we're integrating with Crowdin, it don't like empty strings.